camp/mgetachew/src/main/java/frc/robot/subsystems/MatthewsCoolMotor.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+
+public class MatthewsCoolMotor extends SubsystemBase {
+
+  private WPI_TalonSRX myMotor = new WPI_TalonSRX(Constants.myMotorCanId);
+
+  /** Creates a new RomiDrivetrain. */
+  public MatthewsCoolMotor() {
+  }
+
+  /**
+  * A method exposed to callers to store one ball into storage
```

Comment:
update this comment bloc. - applies to any public method, constructor and class

Reply:
Removed